src/Service/Models/AliasPayload.cs

```diff
@@ -1,10 +1,15 @@
-﻿namespace Passwordless.Service.Models;
+﻿using System.ComponentModel.DataAnnotations;
+using Passwordless.Common.Validation;
 
-using Aliases = HashSet<string>;
+namespace Passwordless.Service.Models;
 
-public class AliasPayload
-{
-    public string UserId { get; set; }
-    public Aliases Aliases { get; set; }
-    public bool Hashing { get; set; } = true;
-}
\ No newline at end of file
+public record AliasPayload
+(
+    [Required(AllowEmptyStrings = false)]
+    string UserId,
```

Comment:
We should consider creating a `UserIdValidation` attribute that encompasses what a valid user id is for the API.  That way its clear that when you have an UserId in the API, it should have a specific shape and that shape will be defined in a single spot.